policyengine_us/variables/gov/states/dc/dhs/tanf/income/dc_tanf_countable_income.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class dc_tanf_countable_income(Variable):
+    value_type = float
+    entity = SPMUnit
+    label = "DC TANF countable income"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.DC
+
+    def formula(spm_unit, period, parameters):
+        gross_earnings = spm_unit("dc_tanf_gross_earned_income", period)
+        gross_unearned_income = spm_unit(
+            "dc_tanf_countable_gross_unearned_income", period
+        )
+        p = parameters(period).gov.states.dc.dhs.tanf.income.earned_deduction
+        annual_flat_exclusion = p.flat * MONTHS_IN_YEAR
+        return (
```

Comment:
To make the code a bit more self-documenting
```suggestion
        earnings_after_flat_exclusion = max_(gross_earnings - annual_flat_exclusion, 0)
        countable_earned_income = earnings_after_flat_exclusion * (1 - p.percentage)
        return countable_earned_income + gross_unearned_income
```